features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/introspect.jag

```diff
@@ -33,31 +34,21 @@
 
     var data = {token: token }
     var introspectResult = post(introspectEndpoint, data , {
-        "Authorization": "Basic YWRtaW46YWRtaW4=", // TODO: get super admin username password from configs and do encoding ~tmkb
+        "Authorization": "Basic " + utils.getBase64EncodedAdminCredentials(),
         "Content-Type": "application/x-www-form-urlencoded"
     });
-
+    introspectResult.data = JSON.parse(introspectResult.data);
     log.debug("Introspection result json: " + introspectResult);
     response.contentType = "application/json";
 
-    var scopes = "apim:api_delete apim:api_view apim:api_create apim:api_publish apim:tier_view apim:tier_manage apim:subscription_view apim:subscription_block apim:mediation_policy_view apim:api_workflow openid"; // TODO: fetch from introspect request
-
     if (introspectResult.data.active) {
```

Comment:
Check the introspect call response status code instead of data.active

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/6667/commits/60ed8577f196f7059f542c18c0b5001939c01ad3